_posts/2021-05-24-indico-3-0-news.md

```diff
@@ -0,0 +1,74 @@
+---
+layout:     post
+title:      "Indico 3.0 🔍"
+date:       2021-05-24 15:00
+author:     "Indico Team"
+categories: indico release
+---
+
+It's with great pleasure that we announce the deployment of **Indico 3.0** at CERN. It is a very important milestone in the history of the Project. Two groundbreaking developments happened during these last months, one of them very visible, while the other one not directly noticeable but nonetheless of vital importance.
```

Comment:
> It's with great pleasure that we announce **Indico 3.0** which will be coming soon.

That way we could already publish the blog post and it'll be more useful when linked in non-CERN places (github release, forum post, etc).

Maybe there's better wording for "which will be coming soon", I just copied it from the previous post :0